src/utils/hooks/useNotifyProjects.ts

```diff
@@ -11,15 +11,17 @@ const useNotifyProjects = () => {
       const explorerApiBaseUrl: string = import.meta.env.VITE_EXPLORER_API_URL
       const projectId: string = import.meta.env.VITE_PROJECT_ID
 
-      const explorerUrl = filterAppDomain? `${explorerApiBaseUrl}/w3i/v1/notify-config?projectId=${projectId}&appDomain=${filterAppDomain}` : `${explorerApiBaseUrl}/w3i/v1/projects?projectId=${projectId}&is_verified=${
-        isDevModeEnabled ? 'false' : 'true'
-      }`
+      const explorerUrl = filterAppDomain
+        ? `${explorerApiBaseUrl}/w3i/v1/notify-config?projectId=${projectId}&appDomain=${filterAppDomain}`
```

Comment:
For better readability, can we use the `URL` class to create URLs? That would help when we add more params or refactor it.


```js
const explorerUrl = new URL(ExplorerEndpoints.projects, explorerApiBaseUrl)
explorerUrl.searchParams.set('projectId', projectId)
explorerUrl.searchParams.set('is_verified', isDevModeEnabled ? 'false' : 'true')
```

Now we have two different options depending on the filterAppDomain so if we could move these URL generation to a utility function

Reply:
Here: https://github.com/WalletConnect/web3inbox/pull/278/files#diff-0d91658008e7f6731408fc6871b1da50e422f43d2e582fe942167fe13a89e732R17-R19

I think I could update the logic here once you merge this one

Reply:
Good point! Yes it's time we refactor some of this